src/main/cpp/src/map_utils.cu

```diff
@@ -112,35 +107,35 @@ rmm::device_uvector<char> unify_json_strings(cudf::column_view const& input,
 }
 
 // Check and throw exception if there is any parsing error.
-void throw_if_error(rmm::device_uvector<char> const& input_json,
-                    rmm::device_uvector<PdaTokenT> const& tokens,
-                    rmm::device_uvector<SymbolOffsetT> const& token_indices,
-                    rmm::cuda_stream_view stream)
-{
+void throw_if_error(rmm::device_uvector<char> const &input_json,
+                    rmm::device_uvector<PdaTokenT> const &tokens,
+                    rmm::device_uvector<SymbolOffsetT> const &token_indices,
+                    rmm::cuda_stream_view stream) {
   auto const error_count =
-    thrust::count(rmm::exec_policy(stream), tokens.begin(), tokens.end(), token_t::ErrorBegin);
+      thrust::count(rmm::exec_policy(stream), tokens.begin(), tokens.end(), token_t::ErrorBegin);
 
   if (error_count > 0) {
     auto const error_location =
-      thrust::find(rmm::exec_policy(stream), tokens.begin(), tokens.end(), token_t::ErrorBegin);
+        thrust::find(rmm::exec_policy(stream), tokens.begin(), tokens.end(), token_t::ErrorBegin);
     SymbolOffsetT error_index;
-    CUDF_CUDA_TRY(
-      cudaMemcpyAsync(&error_index,
-                      token_indices.data() + thrust::distance(tokens.begin(), error_location),
-                      sizeof(SymbolOffsetT),
-                      cudaMemcpyDeviceToHost,
-                      stream.value()));
+    CUDF_CUDA_TRY(cudaMemcpyAsync(
+        &error_index, token_indices.data() + thrust::distance(tokens.begin(), error_location),
+        sizeof(SymbolOffsetT), cudaMemcpyDeviceToHost, stream.value()));
     stream.synchronize();
 
-    constexpr auto extension   = 100;
-    auto const begin_print_idx = std::max(error_index - extension, SymbolOffsetT{0});
+    constexpr SymbolOffsetT extension = 100u;
+
+    // Warning: SymbolOffsetT is unsigned type thus we need to be carefull with subtractions.
+    auto const begin_print_idx =
+        error_index > extension ? error_index - extension : SymbolOffsetT{0};
```

Comment:
The entire file is messed up by the newly upgraded clang-format version from rapids.

This is the only change in this PR.

Reply:
I was about to ask about this.  There seemed to be a *lot* of whitespace changes.  I was having a hard time identifying the location of the fix.